service/slogan.go

```
@@ -2,60 +2,60 @@ package service
+		return fmt.Errorf("更新激励语失败%s", err.Error())
 	}
 	return nil
 }
-
-func (ssr *SloganService) ChangeSlogan(id uint, newSlogan request.Slogan) error {
-
-	var user model.User
-	err := ssr.db.Table("users").Where("id = ?", id).First(&user).Error
+func (ssr *SloganServiceImpl) ChangeSlogan(id uint, newSlogan request.Slogan) error {
+	user, err := ssr.UserDao.GetUserById(id)
 	if err.Error != nil {
```

golangci-lint (1.62.2)

nilfunc: comparison of function Error != nil is always true

(govet)